packages/web/src/pages/upload-page/store/sagas.js

```diff
@@ -698,8 +699,10 @@ function* uploadCollection(tracks, userId, collectionMetadata, isAlbum) {
         console.debug('Creating playlist')
         // Uploaded collections are always public
         const isPrivate = false
+        const tempPlaylistId = getTempPlaylistId()
         const { blockHash, blockNumber, playlistId, error } = yield call(
           audiusBackendInstance.createPlaylist,
+          tempPlaylistId,
```

Comment:
Might be totally misunderstanding this but doesn't this playlist Id become the real id? So maybe we should get rid of the `temp` prefix everywhere?

Reply:
yea that's true for the new flow but it's still actually a temp ID for the legacy playlist flow so i wanted to avoid confusion there. i'll add a comment!